src/components/search/refinement-list.tsx

```diff
@@ -268,6 +268,8 @@ const RefinementList: FunctionComponent<RefinementListProps> = ({
   )
 }
 
+// Docs on how `connectRefinementList` works
+// https://www.algolia.com/doc/api-reference/widgets/refinement-list/react/#connector
```

Comment:
If I'm interpreting this correctly, if we prevent Algolia from indexing the unlisted instructor instance, we won't even have to make changes here, right?

Reply:
this is what we will want to do, but also make sure to associate the listed instructor in the index.

search index switch-a-roo should be our only backend change needed

Reply:
> search index switch-a-roo

this sounds really promising to me. still need to look at exactly what is going into algolia, but I think this could work and make some of the other hackiness go away.

Reply:
Because the Algolia stuff is declarative, it's not immediately clear to me how we can pull this off without letting Algolia index the unlisted instructor and including some form of flag for Algolia to ignore it.

If the connections to the Algolia index are name- or slug-based, then simply ensuring `published?` returns false for unlisted instructors should leave that instructor out of the index, and since it has the same name/slug as the real instructor, Algolia can be oblivious to the fact there's a different one, right?

Reply:
Algolia support conditional indexing https://www.algolia.com/doc/framework-integration/rails/indexing/indexing/?client=ruby#conditional-indexing